inference-engine/tests/functional/shared_test_classes/src/subgraph/mvn_multiply_add.cpp

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "shared_test_classes/subgraph/mvn_multiply_add.hpp"
+
+namespace SubgraphTestsDefinitions {
+
+std::string MVNMultiplyAdd::getTestCaseName(const testing::TestParamInfo<mvnMultiplyAddParams> &obj) {
+    InferenceEngine::SizeVector inputShapes, constantShapes;
+    InferenceEngine::Precision dataPrecision, axesPrecision;
+    std::vector<int> axes;
+    bool normalizeVariance;
+    float eps;
+    std::string epsMode;
+    std::string targetDevice;
+    std::tie(inputShapes, constantShapes, dataPrecision, axesPrecision, axes, normalizeVariance, eps, epsMode, targetDevice) = obj.param;
+    std::ostringstream result;
+    result << "IS=" << CommonTestUtils::vec2str(inputShapes) << "_";
+    result << "CS=" << CommonTestUtils::vec2str(constantShapes) << "_";
+    result << "DataPrc=" << dataPrecision.name() << "_";
+    result << "AxPrc=" << axesPrecision.name() << "_";
+    result << "Ax=" << CommonTestUtils::vec2str(axes) << "_";
+    result << "NormVariance=" << (normalizeVariance ? "TRUE" : "FALSE") << "_";
+    result << "Eps=" << eps << "_";
+    result << "EM=" << epsMode << "_";
+    result << "TargetDevice=" << targetDevice;
+    return result.str();
+}
+
+void MVNMultiplyAdd::SetUp() {
```

Comment:
Cannot we cover this by existing CPU specific MVN test: https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/functional/plugin/cpu/single_layer_tests/mvn.cpp via fusing params?

Reply:
Bug was found in case where fusing shouldn't have been but it was (I think that we cannot catch these cases in CPU specific MVN test).

I added these tests for common subgraph pattern - mvn + mul + add (gamma * mvn + beta). And there should be correct results regardless of whether or not there is a fusing.

Reply:
It's appreciated if you would like add 1D/2D cases in "CPU specific MVN test" as well, w/ or w/o fusing as basic test. I'm sorry to miss that case. Thank you!

Reply:
I added some tests for 1D/2D cases. But I saved subgraph tests because only it can catch this bug

In CPU tests we have MVN with version 0 (has attribute `across_channels`), I created subgraph tests with MVN-6 (has attribute `axis`).

If we have 2D shape and...
- MVN-0 and `across_channels = true`, we won't have fusing ([condition](https://github.com/a-sidorova/openvino/blob/15ffe41ce..)). And cpu specific tests will be failed (we will not be able to find out if the inference results are correct)
- MVN-0 and `across_channels = false`, we will have fusing, but Shape5D won't be changed (like before [PR#5414](https://github.com/openvinotoolkit/openvino/pull/5414)).

In subgraph tests with MVN-6 we can catch some fails regardless of whether or not there is a fusing.

If I'm wrong, correct me please

Reply:
Looks good for me in general. Now in single layer test, `smoke_CompareWithRefs_2D` can cover transformed and no-transformed case w/o fusion.  `smoke_CompareWithRefs_2D_Fuse` can cover no-transformed case w/ fusion. Sub-graph test cover transformed case with fusion. "Transformed " here means shape and acorss_channels is changed for 1[2]D case.